src/omnibar/omnibar-user-activity.ts

```diff
@@ -84,29 +100,29 @@ function startActivityTimer() {
     renewSession();
   }
 
-  intervalId = setInterval(() => {
-    BBOmnibarUserSessionExpiration.getSessionExpiration(
+  intervalId = setInterval(async () => {
+    const expirationDate = await BBOmnibarUserSessionExpiration.getSessionExpiration(
```

Comment:
This didn't turn into a `try/catch` like other refactors. Will that cause a problem here?

Reply:
It won't cause any _new_ problems, since the original code didn't have a `.catch()`. The code would still throw an unhandled Promise exception, but that's what it would have done previously. I'd be inclined to leave it rather than add a `catch` block that does nothing, since there's no logic after it that should complete if it fails.